src/components/admin/admin-settings-tab.yml

```diff
@@ -2,6 +2,24 @@ adminSettingsTab:
   announcementHeader:
     en: Global announcement
     de: Globale Ankündigung
+  announcementInfo:
+    en: This announcement is shown under the website header on all pages. Users can close it and it will only appear again when the text or type are changed.
+    de: This announcement is shown under the website header on all pages. Users can close it and it will only appear again when the text or type are changed.
+  consentHeader:
+    en: Consent text
+    de: Zustimmungstext
+  homepageDocumentsHeader:
+    en: Homepage documents pool
+    de: Homepage-Dokumente-Becken ))
+  homepageDocumentsInfo:
+    en: 3 Documents from this list are randomly selected and displayed on the homepage.
+    de: 3 Documents from this list are randomly selected and displayed on the homepage.
```

Comment:
Aus dieser Liste werden 3 zufällig ausgewählte Dokumente auf der Homepage angezeigt. (I had to reshuffle the words a little bit compared to English)